data-access/Queries/User.cs

```diff
@@ -1,31 +1,64 @@
-﻿using SQE.SqeHttpApi.DataAccess.Models;
+﻿using System.Collections.Generic;
+using System.Linq;
+using SQE.SqeHttpApi.DataAccess.Helpers;
 
 namespace SQE.SqeHttpApi.DataAccess.Queries
 {
-    internal class UserQueryResponse : IQueryResponse<User>
+    internal static class LoginQuery
     {
-        public string user_name { get; set; }
-        public uint user_id { get; set; }
+        public const string GetQuery = @"
+SELECT user_name AS UserName, user_id AS UserId, email AS Email, activated AS Activated
+FROM user 
+WHERE user_name = @UserName AND pw = SHA2(@Password, 224)
+";
+    }
+
+    /// <summary>
+    /// An extensible User info query builder.
+    /// TODO: Upgrade all user queries to use this.
+    /// </summary>
+    internal static class UserDetails
+    {
+        private const string _query = @"
+SELECT $Columns
+FROM user
+WHERE $Where";
 
-        public User CreateModel()
+        /// <summary>
+        /// Returns a formatted query on the user table
+        /// </summary>
+        /// <param name="columns">Names of the columns to be retrieved (in snake_case)</param>
+        /// <param name="where">Names of the where parameters (in snake_case)</param>
+        /// <returns></returns>
+        public static string GetQuery(List<string> columns, List<string> where)
```

Comment:
Now I understand the ToPascalCase. I think this is really overkill. Simply return all the columns and pick the ones you decide to copy to the DTO. We will never face performance issues due to queries of the user table.